### ISSUE TITLE
About softmax

### ISSUE THREAD

**Issue by 321hallelujah** [yuanyao366/PRP#2]
hi,  
when training in self-supervised and fine-tune stage.  your work dont use  softmax after linear layer, which some other related work is used.
So,my question is,: In training,  what is the difference when softmax is applied or not.
Best wishes.

**Comment by yuanyao366** [comment]
We use the "nn.CrossEntropyLoss()" which combines nn.LogSoftmax() and nn.NLLLoss() in one single class.
